File: 프로그래머스/unrated/181926. 수 조작하기 1/수 조작하기 1.swift
func solution(_ n:Int, _ control:String) -> Int {
    var result = n
    control.forEach {
        // print($0)
        switch $0 {
        case "w":
            result += 1
        case "s":
            result -= 1
        case "d":
            result += 10
        case "a":
            result -= 10
        default:
            return
        }
    }
    return result
}

File: 프로그래머스/Lv.0/181926. 수 조작하기 1/수 조작하기 1.js
function solution(n, control) {
    let result = n;
    let dic = {"w":0,"s":0,"d":0,"a":0}
    for(let c of control){
        dic[c] += 1
    }
    
    result += dic['w'] 
    result -= dic['s'] 
    result += (10 * dic['d'])
    result -= (10 * dic['a']) 
    return result;
}